ui/apps/platform/src/Containers/SystemHealth/Components/DeclarativeConfigurationHealthCard.tsx

```diff
@@ -100,10 +100,18 @@ function DeclarativeConfigurationHealthCard({ pollingCount }: CardProps): ReactE
                             <Tbody data-testid="declarative-configs">
                                 {unhealthyItems.map(({ id, name, errorMessage, lastTimestamp }) => (
                                     <Tr key={id}>
-                                        <Td dataLabel="Name" modifier="breakWord">
+                                        <Td
+                                            dataLabel="Name"
+                                            modifier="breakWord"
+                                            data-testid="integration-name"
```

Comment:
Aha, because declarative configuration has a table instead of a list of errors:
1. Element selectors in tests can use `data-label` instead of  `data-testid` attribute.
    Note that React `dataLabel` prop becomes HTML `data-label` attribute.
2. If selector changes from an arbitrary invisible `data-testid` attribute to the a `data-label` attribute that by convention corresponds to the visible table column head text, I recommend replacing the indirection of constants with literal text:
    * Pro: Find in Files according to visible text.
    * Con: Replace if the table column head text changes. Unlikely, and not hard to change, because of Pro.
 
For example,

```js
        cy.get(`${itemSelector} td[data-label="Name"]`).should('have.text', healthName);
        cy.get(`${itemSelector} td[data-label="Error"]`).should('have.text', errorMessageText);
```